lib/config.js

```diff
@@ -83,6 +83,10 @@ const Config = function () {
   this.extraGnArgs = {}
   this.extraNinjaOpts = []
   this.braveSafetyNetApiKey = getNPMConfig(['brave_safetynet_api_key']) || ''
+  this.androidKeystorePath = getNPMConfig(['android_keystore_path']) || ''
+  this.androidKeyAlias = getNPMConfig(['android_key_alias']) || ''
```

Comment:
shouldn't this be `android_keystore_name`?

Reply:
In apksigner this param called `--ks-key-alias`. I don't really care how to name. Will change this.

Reply:
Changed the names of variables